api/src/project_budget_delete_projected.ts

```diff
@@ -133,53 +133,55 @@ export function addHttpHandler(
   urlPrefix: string,
   service: Service,
 ) {
-  server.post(
-    `${urlPrefix}/project.budget.deleteProjected`,
-    mkSwaggerSchema(server),
-    (request, reply) => {
-      const ctx: Ctx = { requestId: request.id, source: "http" };
+  server.register(async function () {
```

Comment:
Maybe the server.register command could be used in index.ts to wrap over all Routes instead of wrapping every route on its own with server.register